### PR TITLE
Minor improvs to hhnet script

### PR DESCRIPTION
**Pull request by edipascale**
* set down interface before enslaving it, to prevent errors if it is already up
* limit the attempts in get_ip to 30 to avoid stalling forever

### CODE REVIEW

File: pkg/hhfab/hhnet.sh [hunk 1]
```diff
@@ -44,13 +46,19 @@ function setup_vlan() {
 function get_ip() {
     local iface_name=$1
     local ip=""
+    local max_attempts=30
```
Comment (Frostman) [inline pkg/hhfab/hhnet.sh:49]: let's set it to a minute just to be on a safe side, thank you

**Review by Frostman — CHANGES_REQUESTED** [review]
thx, it totally makes sense to down interface just in case and not wait forever for IP from DHCP

File: pkg/hhfab/hhnet.sh (hunk 1 above)
Comment (edipascale) [reply pkg/hhfab/hhnet.sh]: done, thanks!

**Comment by mrbojangles3** [comment]
After this passes CI, it looks good to me.

**Review by Frostman — APPROVED** [review]

**Pull request merged**